City-of-Helsinki/drupal-infofinland
# Paragraph settings

Unintalls modules
- paragraphs library
- paragraphs asymmetric translation widgets

Removes possibility to translate Entity reference revisions fields (not recommended by Drupal) and adds possibility to translate paragraph fields.

Changes site default language back to english as it originally was.  


**How to test**

1. Checkout branch
2. run `drush deploy` so config changes are imported
3. Remove existing content as it might not work correctly anymore
4. Create new page and add a text paragraph into it and save
5. Translate the page in english. Change page title in english. Check that is not possible to add new paragraphs. Change the existing text paragraphs text from finnish to english. Save.
6. Go to see the page. Check that it works correctly both in finnish and english and that translations are working correctly.

Anything else related to this error messages etc.?
